main.cc

```diff
@@ -1172,6 +1174,26 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
             scheduling_group_key_config cql_sg_stats_cfg = make_scheduling_group_key_config<cql_transport::cql_sg_stats>(maintenance_socket_enabled::no);
             auto cql_sg_stats_key = scheduling_group_key_create(cql_sg_stats_cfg).get();
 
+            supervisor::notify("starting disk space monitor");
+            auto dsm_cfg = utils::disk_space_monitor::config{
+                .sched_group = dbcfg.compaction_scheduling_group,
```

Comment:
Why compaction?

Seems to me it's a maintenance task.

Reply:
When disk utilization is low this could be done in maintenance (streaming) sched group, but when we're low on disk space, should we monitor the filesystem in higher priority?

Reply:
Hmm, actually, one of the planned use cases is to make streaming shares dynamic, and increase them as we run out of space, we can use the streaming scheduling group all the way.

Reply:
Fixed in v13 ([723785f](https://github.com/scylladb/scylladb/commit/723785f21bd84fa8895d66851d25fc2b19fd915c))